java.lang.ExceptionInInitializerError: null

### Description

![изображение](https://github.com/Imprex-Development/orebfuscator/assets/36813920/951be697-a1f6-4888-89af-f215ba1ecf34)


```
[16:15:58 INFO]: [Orebfuscator] Enabling Orebfuscator v5.4.1

[16:15:59 ERROR]: Error occurred while enabling Orebfuscator v5.4.1 (Is it up to date?)

java.lang.ExceptionInInitializerError: null

	at net.imprex.orebfuscator.obfuscation.ObfuscationSystem.registerChunkListener(ObfuscationSystem.java:39) ~[orebfuscator-plugin-5.4.1.jar:?]

	at net.imprex.orebfuscator.Orebfuscator.onEnable(Orebfuscator.java:82) ~[orebfuscator-plugin-5.4.1.jar:?]

	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:281) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]

	at io.papermc.paper.plugin.manager.PaperPluginInstanceManager.enablePlugin(PaperPluginInstanceManager.java:189) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at io.papermc.paper.plugin.manager.PaperPluginManagerImpl.enablePlugin(PaperPluginManagerImpl.java:104) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:507) ~[purpur-api-1.20.1-R0.1-SNAPSHOT.jar:?]

	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugin(CraftServer.java:654) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at org.bukkit.craftbukkit.v1_20_R1.CraftServer.enablePlugins(CraftServer.java:565) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at net.minecraft.server.MinecraftServer.loadWorld0(MinecraftServer.java:641) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at net.minecraft.server.MinecraftServer.loadLevel(MinecraftServer.java:440) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at net.minecraft.server.dedicated.DedicatedServer.initServer(DedicatedServer.java:345) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at net.minecraft.server.MinecraftServer.runServer(MinecraftServer.java:1117) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:323) ~[purpur-1.20.1.jar:git-Purpur-2044]

	at java.lang.Thread.run(Unknown Source) ~[?:?]

Caused by: java.lang.NullPointerException

	at java.util.Objects.requireNonNull(Unknown Source) ~[?:?]

	at java.util.ImmutableCollections.listFromTrustedArray(Unknown Source) ~[?:?]

	at java.util.List.of(Unknown Source) ~[?:?]

	at net.imprex.orebfuscator.obfuscation.ObfuscationListener.<clinit>(ObfuscationListener.java:30) ~[orebfuscator-plugin-5.4.1.jar:?]

	... 14 more

[16:15:59 INFO]: [Orebfuscator] Disabling Orebfuscator v5.4.1

[16:15:59 ERROR]: Error occurred (in the plugin loader) while disabling Orebfuscator v5.4.1 (Is it up to date?)

```
### Reproduction Steps

Install ProtocolLib and Orebfuscator

### Expected Behaviour

work

### Orebfuscator Dump

Impossible, plugin not working

### Extra Details

_No response_

--- a/orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/obfuscation/ObfuscationListener.java
+++ b/orebfuscator-plugin/src/main/java/net/imprex/orebfuscator/obfuscation/ObfuscationListener.java
@@ -1,5 +1,6 @@
 package net.imprex.orebfuscator.obfuscation;
 
+import java.util.Arrays;
 import java.util.List;
 import java.util.Objects;
 import java.util.Set;
@@ -27,7 +28,7 @@ import net.imprex.orebfuscator.util.PermissionUtil;
 
 public abstract class ObfuscationListener extends PacketAdapter {
 
-	private static final List<PacketType> PACKET_TYPES = List.of(
+	private static final List<PacketType> PACKET_TYPES = Arrays.asList(
 		PacketType.Play.Server.MAP_CHUNK,
 		PacketType.Play.Server.UNLOAD_CHUNK,
 		PacketType.Play.Server.LIGHT_UPDATE,

